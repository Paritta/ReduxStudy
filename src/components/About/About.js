import React from "react";
import styled from "styled-components";
import oc from "open-color";

const propTypes = {
};

const defaultTypes = {
};

const Wrapper = styled.div`
    background: ${oc.gray[1]};
    height: 90vh;
    
    display: flex;
    flex-direction: column;
`;

const Footer = styled.div`
    background: ${oc.gray[1]};
    height: 10vh;
`;

const Face = styled.div`
    background: white;
    height: 100px;
    
    width: 100px;
    height: 10vh;
`;

const Body = styled.div`
    background: white;
    height: 10vh;
    
    width: 100px;
    height: 10vh;
`;

export const About = () => {
    return (
        <div>
            <Wrapper>
                About
            </Wrapper>
            <Footer>
                친구들
            </Footer>
        </div>
    )
};

About.propTypes = propTypes;
About.defaultTypes = defaultTypes;

export default About;
