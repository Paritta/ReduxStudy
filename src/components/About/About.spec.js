import React from "react";
import { About } from "./About";
import "jest-styled-components";
import ReactTestRenderer from "react-test-renderer";

it("Card render correctly", () => {
    const tree = ReactTestRenderer.create(
        <About />
    );
    expect(tree.toJSON()).toMatchSnapshot();
});
