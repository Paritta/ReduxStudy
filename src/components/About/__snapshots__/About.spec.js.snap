// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render correctly 1`] = `
.c0 {
  background: #f1f3f5;
  height: 90vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 {
  background: #f1f3f5;
  height: 10vh;
}

<div>
  <div
    className="c0"
  >
    About
  </div>
  <div
    className="c1"
  >
    친구들
  </div>
</div>
`;
